Bottom: 030710037ff423b76f8ff8badf1ba0d9135e98db
Top:    c52933538d6cf22b5bf4ada59eabbf94db09a3bf
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 15:36:35 -0500

Refresh of gitlab-auth

---
diff --git a/ansible-playbook/group_vars/all/credentials.yml b/ansible-playbook/group_vars/all/credentials.yml
index d476bf91..828dde00 100644
--- a/ansible-playbook/group_vars/all/credentials.yml
+++ b/ansible-playbook/group_vars/all/credentials.yml
@@ -5,6 +5,8 @@ buildsystem_database_build_password: '{{ lookup("password", playbook_dir + "/cre
 buildsystem_database_root_password:  '{{ lookup("password", playbook_dir + "/credentials/buildsystem/database/root"           , length=20) }}'
 buildsystem_ssh_prod_user_password:  '{{ lookup("password", playbook_dir + "/credentials/buildsystem/ssh/prod"                , length=20) }}'
 buildsystem_ssh_root_user_password:  '{{ lookup("password", playbook_dir + "/credentials/buildsystem/ssh/root"                , length=20) }}'
+gitlab_buildbot_auth_id:             '{{ lookup("password", playbook_dir + "/credentials/git/auth/buildbot_client_id") }}'
+gitlab_buildbot_auth_secret:         '{{ lookup("password", playbook_dir + "/credentials/git/auth/buildbot_secret") }}'
 gitlab_db_password:                  '{{ lookup("password", playbook_dir + "/credentials/git/db/user"                         , length=20) }}'
 gitlab_email_password:               '{{ lookup("password", playbook_dir + "/credentials/git/email/gitlab@novatech-llc.com"   , length=20) }}'
 gitlab_root_password:                '{{ lookup("password", playbook_dir + "/credentials/git/users/root"                      , length=20) }}'
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 3adede49..c9a90fab 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -44,6 +44,12 @@ container_env:
   BUILDBOT_WORKER_NTEL_PASS: '{{ buildbot_worker_ntel_password }}'
   BUILDBOT_WORKER_PTXDIST: 'worker-ptxdist'
   BUILDBOT_WORKER_PTXDIST_PASS: '{{ buildbot_worker_ptxdist_password }}'
+  GITLAB_URI: 'https://{{ gitlab_hostname }}/'
+  GITLAB_CLIENT_ID: '{{ gitlab_buildbot_id }}'
+  GITLAB_CLIENT_SECRET: '{{ gitlab_buildbot_secret }}'
+  LDAP_URI: 'ldap://{{ openldap_hostname }}'
+  LDAP_BIND_USER: 'cn=proxyagent,dc=novatech'
+  LDAP_BIND_PW: '{{ openldap_proxyagent_password }}'
 
 # the name of the container being started
 container_name: '{{ docker_name_prefix }}buildbot'
